3 tests fail while running pytest

Hi All,

   are all tests in the current version supposed to be green? 3 tests fail while running `pytest` (I'm using GeForce GTX TitanX):

```
FAILED tests/test_wrappers.py::test_conv2d - RuntimeError: element 0 of tensors does not require grad...
FAILED tests/test_wrappers.py::test_conv_transposed_2d - RuntimeError: element 0 of tensors does not ...
FAILED tests/test_wrappers.py::test_linear - RuntimeError: element 0 of tensors does not require grad...

```
with the same error message:

`RuntimeError: element 0 of tensors does not require grad and does not have a grad_fn`

any ideas?

Thanks in advance for your help!

Best,
Alexey

--- a/tests/test_async.py
+++ b/tests/test_async.py
@@ -77,7 +77,6 @@ class AsyncInferenceTestCase(AsyncTestCase):
             img_path = os.path.join(root_dir, 'demo/demo.jpg')
             bboxes, _ = await detector.apredict(img_path)
             self.assertTrue(bboxes)
-            torch.set_grad_enabled(ori_grad_enabled)
             # asy inference detector will hack grad_enabled,
             # so restore here to avoid it to influence other tests
             torch.set_grad_enabled(ori_grad_enabled)

--- a/tests/test_async.py
+++ b/tests/test_async.py
@@ -68,6 +68,7 @@ class AsyncInferenceTestCase(AsyncTestCase):
 
                 pytest.skip('test requires GPU and torch+cuda')
 
+            ori_grad_enabled = torch.is_grad_enabled()
             root_dir = os.path.dirname(os.path.dirname(__name__))
             model_config = os.path.join(
                 root_dir, 'configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py')
@@ -76,3 +77,7 @@ class AsyncInferenceTestCase(AsyncTestCase):
             img_path = os.path.join(root_dir, 'demo/demo.jpg')
             bboxes, _ = await detector.apredict(img_path)
             self.assertTrue(bboxes)
+            torch.set_grad_enabled(ori_grad_enabled)
+            # asy inference detector will hack grad_enabled,
+            # so restore here to avoid it to influence other tests
+            torch.set_grad_enabled(ori_grad_enabled)

--- a/tests/test_heads.py
+++ b/tests/test_heads.py
@@ -230,7 +230,7 @@ def test_ga_anchor_head_loss():
         gt_bboxes = [
             torch.Tensor([[23.6667, 23.8757, 238.6326, 151.8874]]).cuda(),
         ]
-        gt_labels = [torch.LongTensor([2])]
+        gt_labels = [torch.LongTensor([2]).cuda()]
         one_gt_losses = head.loss(cls_scores, bbox_preds, shape_preds,
                                   loc_preds, gt_bboxes, gt_labels, img_metas,
                                   gt_bboxes_ignore)

